shards/modules/inputs/inputs.cpp

```diff
@@ -620,4 +668,6 @@ SHARDS_REGISTER_FN(inputs) {
   using KeyUp = KeyUpDown<false>;
   REGISTER_SHARD("Inputs.KeyDown", KeyDown);
   REGISTER_SHARD("Inputs.KeyUp", KeyUp);
+
+  REGISTER_SHARD("Inputs.MatchModifierKeys", MatchModifierKeys);
```

Comment:
A bit long, `MatchModifier` should be enough , we already have `Input` as context

Reply:
Ok